ravenpy/cli.py

```diff
@@ -34,13 +34,23 @@ def cli():
     type=click.Tuple([str, str]),
     default=RoutingProductGridWeightImporter.VAR_NAMES,
     show_default=True,
-    help="Ordered variable names of longitude (x) and latitude (y) in the NetCDF file.",
+    help="(A) Variable name of 2D longitude and latitude variables in NetCDF (in this order). Example: 'lon,lat'. Or (B) Attribute name in input_file shapefile (option -i) that defines the index of the shape in NetCDF model output file (numbering needs to be [0 ... N-1]).",
```

Comment:
Revert to previous version as we now have the `routing-id-field` as separate option. Please leave "longitude (x)"  and "latitude (y)". It's easier for some people to refer to x/y.